engine/install/debian.md

```diff
@@ -22,8 +22,8 @@ To get started with Docker Engine on Debian, make sure you
 To install Docker Engine, you need the 64-bit version of one of these Debian or
 Raspbian versions:
```

Comment:
```suggestion
To install Docker Engine, you need the 64-bit version of one of these Debian versions:
```

Reply:
Oh, actually, I should add Raspbian back for these 🤦